FabricASEngineAnalytics/README.md

```diff
@@ -10,21 +10,88 @@ The following parameters are defined in the template:
+|Fabric Item Id |Optionally enter a fabric artifact (semantic model) Id. |
 
 ![Screenshot of the Edit Parameters dialog.](./media/parameters.png)
 
 ## Report
 
+### Data Model
+The model has the following tables and the relationships:
+* Calendar
+  * 30s granularity.
+  * Has columns like Date and Hour feed to the "Filters on all pages" section.
+* Artifact
+  * This is a calculated table based on the ExecutionMetrics table.
+  * Has columns like WorkspaceId, WorkspaceName, ArtifactId/Semantic Model Id, ArtifactName/Semantic Model Name feed to the "Filters on all pages" section.
+* ExecutionMetrics
+  * This is a list of all the XmlaRequestId that has ExecutionMetrics and can associated with a semantic moded. Plus, the XmlaRequesstId can associated with a semantic model but no ExecutionMetrics (on-going refreshes or missing traces).
+  * Has many to one relationshop to the Calendar and Artifiact tables.
+  * Has the columns like CapacityId, XmlaRequestId, ApplicationName and EffectiveClaims/Identity feed to the "Filters on all pages" section.
+* Command
+  * This is a list of the XmalRequestId for data refreshes, as well as commands like backup, restore, deployment using the XMLA endpoint.
+  * Has one to one relatinship to the ExecutionMetrics table.
+  * Has the columns like OperationDetailName, RequestApp, RequestID, Status, StatusCode, CommandHash feed to the "Filter on this page" of the "Command/Refresh" page.
+* Query
+  * This is a list of the XmalRequestId for user queries.
+  * Has one to one relationship to the ExecutionMetrics table.
+  * Has the columns like OperationDetailName, RequestApp, RequestID, Status, StatusCode, QueryHash, DatasetId, ReportId, VisualId, DurationBucket, DatasetMode, ReplicaId feed to the "Filter on this page" of the "Query" page.
+* Progress Report
+  * Details of the data refresh at the table and partition level.
+  * Has many to one relationship to the Command table.
+  * Feeds to the "Command/Refresh" page and the Drill through page: "Refresh details".
+   ![image](https://github.com/microsoft/PowerBI-LogAnalytics-Template-Reports/assets/167439901/3b671578-dfec-42bf-a734-d9ecf80f9564)
+
 ### Page: Overview
 
-TODO: Brief detail of each page, how to interact and type of analysis
+* Execution
+  *  Dril down the chart to 30s granuality.
+  *  From the Card visual Drill through to the "Query" or "Command/Refresh" page.
+  *  From the Table visual Drill through to the "Query" or "Command/Refresh" page carrying the semantic model as a filter.
+  ![image](https://github.com/microsoft/PowerBI-LogAnalytics-Template-Reports/assets/167439901/13678800-77c3-41c5-947e-74aebbba1308)
+
+* Command/Refresh
+  * For scheculed and on-demand refresh, the RequestID is coresponding to the Reques ID in the refresh history which can have multiple re-tries.
+      
+       ![image](https://github.com/microsoft/PowerBI-LogAnalytics-Template-Reports/assets/167439901/722789f5-5858-4d5b-aa4b-b275b27c65e0)
```

Comment:
Please add all images to the \media folder inside of \FabricASEngineAnalytics and refer to it with a relative path.